Vagrantfile

```diff
@@ -181,7 +192,7 @@ scissor = {
     :autostart => false,
     :box => CENTOS_BOX_ID,
     :cpus => 1,
-    :mac_address => "0800271F9D13",
+    :mac_address => "0800271F9D14",
```

Comment:
Don't change MAC addresses. They are used to statically assign IP addresses via DHCP.